packages/scripts/utils/config.js

```diff
@@ -344,16 +342,6 @@ function getWebpackEntryPoints( buildType ) {
 					entryPoints[ entryName ] = entryFilepath;
 				}
 			}
-
-			if ( Object.keys( entryPoints ).length > 0 ) {
-				return entryPoints;
-			}
-		}
-
-		// Don't do any further processing if this is a module build.
```

Comment:
Why was it returning early with no entries for module builds? The comment doesn't provide context unfortunately.